src/lib/custom-pointer-events/mouse-move.js

```diff
@@ -0,0 +1,40 @@
+import PointerEvt from '@/lib/custom-pointer-events/pointer-evt.js'
+
+export default class MouseMove extends PointerEvt {
+  constructor (element, evtHandler) {
+    super()
+    this.evtType = 'mousemove'
+    this.element = element
+    this.evtHandler = evtHandler // A client's event handler
+    this.boundListener = this.eventListener.bind(this) // A bound event listener of MouseDblClick
+    this.mouseMoveTimeout = null
+    this.mouseMoveDelay = 1000

```

Comment:
it might be good to make the mouseMoveDelay a configurable setting

Reply:
Done